src/dproto_tcp.erl

```diff
@@ -25,8 +25,12 @@
 -export_type([ttl/0, bucket_info/0, tcp_message/0,
               batch_message/0, stream_message/0]).
 
+
 -type ttl() :: pos_integer() | infinity.
 
+-type read_opts() :: [{rr, default} | {rr, off} | {rr, on} |
+                      {r, n} | {r, default} | {r, pos_integer()}].
```

Comment:
A extracted read_opt type might be nicer to read then a list with or's in it (I've been guilty of that myself :( ). With more complex types like this 1 alternative per line usually reads better a well.

`pos_integer()` should probably be `1..254` to exclude 255.